test/book_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/book'
+
+class BookTest < Minitest::Test
+  def setup
+    @book = Book.new({
+      author_first_name: "Harper",
+      author_last_name: "Lee",
+      title: "To Kill a Mockingbird",
+      publication_date: "July 11, 1960"})
+  end
+
+  def test_it_exists
+    assert_instance_of Book, @book
+  end
+
+  def test_attributes_at_init
+    assert_equal "To Kill a Mockingbird", @book.title
+    assert_equal "Harper Lee", @book.author
+    assert_equal 1960, @book.publication_year
```

Comment:
A small thing, but the interaction pattern indicates that this should be the string `"1960"`